packages/urql/updates/index.js

```diff
@@ -1,90 +1,129 @@
-import createComment from './createComment'
+import { get } from 'lodash/fp'
+import meQuery from '@hylo/graphql/queries/meQuery'
 import meCheckAuthQuery from '@hylo/graphql/queries/meCheckAuthQuery'
+import makeAppendToPaginatedSetResolver from './makeAppendToPaginatedSetResolver'
 
 export default {
   Mutation: {
-    addProposalVote: (result, args, cache, info) => { 
+    addProposalVote: (result, args, cache, info) => {
+      // TODO: URQL - This will result in Me being fully re-queried with every new skilled added, which may be fine...
+      // but probably can apply makeAppendToPaginatedSetResolver here as it is a QuerySet
       cache.invalidate({ __typename: 'Post', id: args.postId })
     },
+
     addSkill: (result, args, cache, info) => {
+      // TODO: URQL - This will result in Me being fully re-queried with every new skilled added, which may be fine...
+      // but probably can apply makeAppendToPaginatedSetResolver here as it is a QuerySet
       cache.invalidate('Query', 'me')
     },
+
     clearModerationAction: (result, args, cache, info) => {
       if (result[info.fieldName].success) {
         cache.invalidate({ __typename: 'ModerationAction', id: args.moderationActionId })
       }
     },
-    createComment,
+
+    createComment: (result, args, cache, info) => {
+      const parentId = args?.data?.parentCommentId || args?.data?.postId
+      const parentType = args?.data?.parentCommentId ? 'Comment' : 'Post'
+      const fieldName = args?.data?.parentCommentId ? 'childComments' : 'comments'
+      makeAppendToPaginatedSetResolver({
```

Comment:
So this is the general pattern for working with our abundance of nexted querySets?

Reply:
Exactly, but only where needed, it isn't always/usually.